server/controllers/auth.js

```diff
@@ -1,70 +1,52 @@
-(() => {
-  'use strict';
+import jwt from 'jsonwebtoken';
+import helper from './../../services/helpers';
 
-  const jwt = require('jsonwebtoken');
-  const helper = require('./../../services/helpers');
-  const secret = process.env.secret ;
+const secret = process.env.secret || 'secret';
 
-  const Auth = {
+/**
+* Provies methods to handle authentication
+*
+* @return {void}
```

Comment:
### Critical

Classes `NEVER` return anything, so this line should not show up for any class definition.